lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -70,40 +73,88 @@ def initialize(serializer: Serialize, **rest)
       # An event was triggered.
       # Re-evaluate all subscribed queries and push the data over ActionCable.
       def execute_all(event, object)
+        execute_legacy(event, object) if config.handle_legacy_subscriptions
+
+        fingerprints = redis.zrevrange(FINGERPRINTS_PREFIX + event.topic, 0, -1)
```

Comment:
As per Redis 6.2.0, this command is considered deprecated. Please prefer using the ZRANGE command with the REV argument in new code.

Reply:
Redis 6.2 with these new arguments was released less than a month ago. And there is no support for these arguments in Ruby client for Redis yet: https://github.com/redis/redis-rb/blob/05425d4d51287e00b5187a75885c45ac7172bacb/lib/redis.rb#L1775

However, we can just use `zrange` as there is no real difference in which order we will traverse fingerprints: starting from ones with most subscriptions down to least (with `zrevrange`), or starting from with least subscriptions up to most (with `zrange`).